modules/apps/social/social-bookmarks-taglib/src/main/resources/META-INF/resources/bookmark/deprecated_bookmark.jsp

```diff
@@ -17,7 +17,7 @@
 <%@ include file="/bookmark/init.jsp" %>
 
 <%
-String icon = PropsUtil.get(PropsKeys.SOCIAL_BOOKMARK_ICON, new Filter(type));
+String icon = PropsUtil.get("social.bookmark.icon", new Filter(type));
```

Comment:
@dantewang @yuhai 
Please note I did not include this commit. Reason is this is not a real usage removal, actually it is a property inline, so let's keep it for now.

Tina.